### ISSUE TITLE
Way to customise colors

### ISSUE THREAD

**Issue by KasunHasanga** [chandrabezzo/CountryCodePicker#8]
In these days most app use dark theme or some custome theme, Please make this compatible to more customise to developer

**Comment by zeeshanahmad0201** [comment]
Yeah, it would be amazing if the dialog color is based on the current app theme. Currently, it is like ![this](https://github.com/chandrabezzo/CountryCodePicker/assets/50322893/eab9080b-4091-491e-9774-855a6304e67a)
